### DISCUSSION TITLE
Does GitHub Pages support Node.js

### DISCUSSION THREAD

**Discussion by murxg** [community/community#22507] [Other Feature Feedback, Questions, & Ideas]
<p>So does Github Pages support Node.js and how to add it without any application for coding and terminal? :thinking:</p>

**Comment by yoannchaudet** [comment]
<p>Hello,</p>
<p>We do not. Pages is a static site hosting solution only today.</p>

**Comment by murxg** [comment]
<p>Ok! Thank you for information!</p>
